etc/samples/tendrl_definitions_generic-0.1.sample.yaml

```diff
@@ -0,0 +1,77 @@
+---
+valid_objects:
+- volume
+
+object_details:
+  volume:
+    enabled: True
+    value: "/clusters/gluster/$cluster.id/volumes/$volume.id
+    attrs:
+      volname:
+        type: String
+        help: "Name of the gluster volume"
+      stripe_count:
+        type: Integer
+        help: "Volume Stripe count"
+      replica_count:
+        type: Integer
+      disperse_count:
+        type: Integer
+      redundancy_count:
+        type: Integer
+      transport:
+        type: String
+      brickdetails:
+        type: Brick[]
+      force:
+        type: Boolean
+      directory:
+        type: String
+      bitrot:
```

Comment:
Fields specific to bitrot, quota etc might not be required to be maintained at the moment. They could be added as and when features get added to tendrl I feel. What other think on this ?